helm/wireguard/values.yaml

```diff
@@ -114,3 +114,86 @@ nodeSelector: {}
 tolerations: []
   # - effect: NoSchedule
   #   operator: Exists
+
+## Metrics configuration
+metrics:
+  # -- Enable exposing Wireguard metrics
+  enabled: true
```

Comment:
Can you switch this to be disabled by default so people can opt into enabling this?

This is causing a failure in CI because it relies on the following CRD being loaded into the cluster.

``` yaml
apiVersion: monitoring.coreos.com/v1
 kind: ServiceMonitor
```